test/ArrayField_test.js

```diff
@@ -38,21 +38,21 @@ describe("ArrayField", () => {
       expect(legend.id).eql("root__title");
     });
 
-    it("should contain no field in the list by default", () => {
+    it("should contain no field in the list by default", function*() {
```

Comment:
Side notes:

- this may be highly fragile as it probably heavily depends on the current Babel polyfill implementation details;
- we should also look into `mocha` code to see if they're doing anything specific when generator functions are passed to `it()`.